### ISSUE TITLE
missing weekday column in calendar.txt can crash parsing/validation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#23]
```
This is what the trace looks like:

Traceback (most recent call last):
  File "./feedvalidator.py", line 64, in ?
    loader.Load()
  File "/home/jhughes/gtdf/trunk/python/transitfeed.py", line 2271, in Load
    self._LoadCalendar()
  File "/home/jhughes/gtdf/trunk/python/transitfeed.py", line 2049, in
_LoadCalendar
    period = ServicePeriod(field_list=row)
  File "/home/jhughes/gtdf/trunk/python/transitfeed.py", line 1203, in __init__
    self.original_day_values += [value.strip()]
AttributeError: 'NoneType' object has no attribute 'strip'

```

Original issue reported on code.google.com by `joe.hugh...@gmail.com` on 17 May 2007 at 4:41

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 1.0.7
```

Original comment by `joe.hugh...@gmail.com` on 21 May 2007 at 4:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `joe.hugh...@gmail.com` on 21 May 2007 at 4:57
- Changed state: **Fixed**
